bool remote_update
---
ExchangeApp[] installed_apps
ExchangeApp[] available_apps